helm/app-operator/templates/configmap.yaml

```diff
@@ -17,7 +17,11 @@ data:
       app:
         unique: {{ include "resource.app.unique" . }}
       chart:
+        {{- if hasSuffix "-unique" .Release.Name }}
         webhookBaseURL: 'http://{{ include "resource.default.name" . }}.{{ include "resource.default.namespace" . }}.svc.cluster.local:{{ .Values.port }}'
+        {{- else }}
+        webhookBaseURL: 'http://{{ include "resource.default.name" . }}.{{ .Values.Installation.V1.Kubernetes.API.Address }}'
```

Comment:
If the release name is `app-operator-dev` in `gauss` installation, this would be `app-operator-dev.g8s.gauss.eu-west-1.aws.gigantic.io`

Reply:
So we use the service for the instance, like `app-operator-2-3-5`?  I think that's the right approach. 

The TLS handling looks good but can we also use IP whitelisting? So only TCs can connect?

Reply:
Do you mean annotation `nginx.ingress.kubernetes.io/whitelist-source-range:`? but how do we set the IP range only for TCs?

Reply:
Okay, @corest confirmed that this is not possible since tenant clusters in AWS are using NAT gateway, which is dynamic, so we can't set the whitelist as upfront.

Reply:
Yes whitelist isn't going to work. Maybe we can use an auth token? Let's look into that next week.